libc/src/__support/OSUtil/windows/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+add_object_library(
+  windows_util
+  SRCS
+    exit.cpp
+    io.cpp
+  HDRS
+    io.h
+  DEPENDS
```

Comment:
you can add `libc.src.__support.macros.config` to avoid empty `DEPENDS`.

Reply:
Updated.